salt/pillar/__init__.py

```diff
@@ -260,40 +272,24 @@ class Pillar(object):
     '''
```

Comment:
Most everything above are pylint improvements.  This is where the changes to pillar rending happen.